modules/openshift-cluster-maximums-environment.adoc

```diff
@@ -56,3 +56,51 @@ AWS cloud platform:
 4. Larger disk size is used so that there is enough space to store the large amounts of data that is collected during the performance and scalability test run.
 5. Cluster is scaled in iterations and performance and scalability tests are executed at the specified node counts.
 --
+
+IBM Power Systems platform:
+
+[options="header",cols="7*"]
+|===
+| Node |Flavor |vCPU |RAM(GiB) |Disk type|Disk size(GiB)/IOS |Count
+
+| Master/etcd ^[1]^
+| r5.4xlarge
```

Comment:
I do not know where these flavor names are coming from.  Can we drop this whole Flavor Column?

Reply:
@manojnkumar  I have replaced r5.4xlarge in flavor column with P9-e980 . That was mistake on my part. Hope this is fine, if you still want this column to be removed I can.

Reply:
> @manojnkumar I have replaced r5.4xlarge in flavor column with P9-e980 . That was mistake on my part. Hope this is fine, if you still want this column to be removed I can.

@ktania46 : Yes, I would prefer that we remove it.